README.md

```diff
@@ -23,6 +24,20 @@ Supported Python version is >= **3.7**.
 pip install m2cgen
 ```
 
+## Development
+The following system dependencies are required for API tests to run successfully:
+* gcc
+* cmake
+* libomp
```

Comment:
Sorry, I can't find `cmake` and `libomp` in [our Dockerfile](https://github.com/BayesWitnesses/m2cgen/blob/master/Dockerfile). I guess they aren't required to run tests (especially, API ones).

Reply:
Ah, seems like this is Mac OS specific. Let me clarify this in the README.

Reply:
Btw, all of these dependencies were needed to compile LightGBM during its installation.

Reply:
For example I was getting an error like:
```
         raise Exception("\n".join((error_msg, LOG_NOTICE)))
      Exception: Please install CMake and all required dependencies first
      The full version of error log was saved into /Users/izeigerman/LightGBM_compilation.log
      [end of output]
 ```
 and further in compilation logs:
 ```
$ cat /Users/izeigerman/LightGBM_compilation.log
-- The C compiler identification is AppleClang 13.1.6.13160021
-- The CXX compiler identification is AppleClang 13.1.6.13160021
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at /opt/homebrew/Cellar/cmake/3.24.2/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find OpenMP_C (missing: OpenMP_C_FLAGS OpenMP_C_LIB_NAMES)
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.24.2/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /opt/homebrew/Cellar/cmake/3.24.2/share/cmake/Modules/FindOpenMP.cmake:545 (find_package_handle_standard_args)
  CMakeLists.txt:135 (find_package)
 ```
 Please note that I tried this on Mac with the M1 chip.

Reply:
Deleted this block altogether for now since it's platform-specific.

Reply:
Ah, you are right! Unfortunately, LightGBM doesn't provide wheels for macOS ARM architecture. This results in installation from sources when you `pip install lightgbm`.